.gitlab/package_build/windows.yml

```diff
@@ -28,6 +29,8 @@
     - remove-item -recurse -force build-out\${CI_JOB_ID}
     - get-childitem build-out
     - get-childitem .omnibus\pkg
+  after_script:
+    - '$_instance_id = (iwr  -UseBasicParsing http://169.254.169.254/latest/meta-data/instance-id).content ; Write-Host "Running on instance $($_instance_id)"'
```

Comment:
Is there a specific reason to also log it in the `after_script` here? Is it because the job log is too long?

Reply:
yeah, just make it easier to find.
it's not 100% necessary